docs/DeveloperGuide.md

```diff
@@ -1210,4 +1210,17 @@ i. Click on the folder that the jar file had been saved in.
 ii. Select both the 'tasks.txt' and 'notebooks.txt'.
 iii. Delete both files.  
 iv. Restart the application by double-clicking the jar file and running Zer0Note.  
-Expected: The Command Line Interface should launch with a welcome note from Zer0Note as shown in Appendix F, 1.1.  
+Expected: The Command Line Interface should launch with a welcome note from Zer0Note as shown in Appendix F, 1.1. 
+
+3.2 Dealing with corrupted name file  
+i. Click on the folder that the jar file had been saved in.  
+ii. Enter the 'src', then 'main', 'resources' and then 'txt' folder.  
```

Comment:
wait, I'm not sure if the user has access to src... o.o

Reply:
Oh wait they arent doing this?

![image](https://user-images.githubusercontent.com/57039314/97823756-d7059f80-1cf4-11eb-97ac-54f8503357af.png)

Reply:
OH RIGHT THIS IS THE DG AND NOT UG SORRY